Earthfile

```diff
@@ -106,42 +115,30 @@ go-deps-test:
     SAVE ARTIFACT go.mod go.mod AS LOCAL go.mod
     SAVE ARTIFACT go.sum go.sum AS LOCAL go.sum
 
-CONTAINER_IMAGE_VERSION:
-  COMMAND
-
-  ARG VERSION
-
-  # quay.io doesn't accept "+" in the repo name
-  ARG _VERSION=$(echo $VERSION | sed 's/+/-/')
-
-  IF [ "$IMAGE" = "" ]
-    # TODO: This IF block should be deleted as soon as our repository names
-    # follow our conventions.
-    IF [ "$VARIANT" = "standard" ]
-      RUN echo ${BASE_URL}/kairos-${FLAVOR}:${_VERSION} > IMAGE
-    ELSE
-      RUN echo ${BASE_URL}/${VARIANT}-${FLAVOR}:${_VERSION} > IMAGE
-    END
-  ELSE
-    RUN echo $IMAGE > IMAGE
-  END
-
-  RUN echo "${VERSION}" > VERSION
 
 OSRELEASE:
     COMMAND
-    ARG OS_ID
-    ARG OS_NAME
-    ARG OS_REPO
-    ARG OS_VERSION
-    ARG VARIANT
-    ARG FLAVOR
     ARG GITHUB_REPO
     ARG BUG_REPORT_URL
     ARG HOME_URL
 
-    COPY +version/VERSION ./
-    ARG OS_LABEL=$(cat VERSION)
+    ARG OS_ID=kairos
+
+    # For naming.sh
+    ARG TARGETARCH # Earthly built-in (not passed)
+    ARG --required FAMILY
+    ARG --required FLAVOR
+    ARG --required FLAVOR_RELEASE
+    ARG --required VARIANT
+    ARG --required MODEL
+    ARG --required KAIROS_VERSION
+
+    COPY ./naming.sh .
```

Comment:
should we have a target like the VERSION instead of copying stuff around? or a COMMAND?

Reply:
We could, yes. I'm hoping we'll have less earthly in the next iteration though. Should we see how it looks after this: https://github.com/kairos-io/kairos/issues/1897 and decide then?